src/echarts.js

```diff
@@ -1142,8 +1142,22 @@ echartsProto.resize = function (opts) {
     flushPendingActions.call(this, silent);
 
     triggerUpdatedEvent.call(this, silent);
+
+    hideTooltip.call(this);
 };
 
+/**
+ * when alwaysShowContent is true change or rotation window size and restore will hide tooltip
+ */
+function hideTooltip() {
+    const tooltips = this._componentsViews.filter(item => item.__model.mainType === 'tooltip');
```

Comment:
Please use ES3, so `const` and `filter` are not allowed to be used in this project
https://echarts.apache.org/zh/coding-standard.html#language-features